src/Code.gs

```diff
@@ -607,15 +630,20 @@ FirebaseApp_._sendAllRequests = function (finalRequests, originalsRequests, db,
       continue;
     }
     
-    // mainly "Permission denied" error, can also be "Invalid path" error
-    else if (responseCode === 401) {
-      originalsRequests[i].error = new Error(responseParsed.error || FirebaseApp_._PERMISSION_DENIED);
+    if (responseCode === 401) {
+      originalsRequests[i].error = new Error(responseParsed.error || FirebaseApp_.NORMALIZED_ERRORS.PERMISSION_DENIED);
+      
+      continue;
+    }
+    
+    if (responseParsed && responseParsed.error && FirebaseApp_.NORETRY_ERRORS[responseParsed.error]) {
+      originalsRequests[i].error = new Error(responseParsed.error);
       
       continue;
     }
     
-    // All other cases are errors that we do not retry
-    originalsRequests[i].error = new Error(FirebaseApp_._ERROR_TRY_AGAIN);
+    // All other cases are errors on which we perform a retry
```

Comment:
nope ! Retries are taken care of above:
`FirebaseApp_._errorCodeList[responseCode] || errorMessage`

At this point these are really errors not re-tried, so we return an error to the user, (though the erreur message is not helpful, that was the original message I think)...